docs/changes/units/13638.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Unit changes of the form ``quantity <<= new_unit`` will now ensure that the
```

Comment:
The bugfix message is not quite right anymore. Maybe just `Unit changes of the form ``quantity <<= new_unit`` will now work also if the quantity is integer. The result will always be float. This means...`

Reply:
👍